owrx/controllers/template.py

```diff
@@ -42,4 +43,27 @@ def indexAction(self):
 class MapController(WebpageController):
     def indexAction(self):
         # TODO check if we have a google maps api key first?
-        self.serve_template("map.html", **self.template_variables())
+        # self.serve_template("map.html", **self.template_variables()) # AF: to be removed once the PR is accepted.
+        self.serve_template("map-%s.html" % self.map_type(), **self.template_variables())
```

Comment:
i believe in general, `str.format()` should be preferred over the python-2 style `%` templating.

Reply:
Anything you like. I personally have no preference.

Reply:
ported the change from Marat's repo.